css-media-queries/breakpoints.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Media Queries - Breakpoints</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="pokemon.css">
+  <style>
+
+    #card-deck {
+      display: flex;
+      flex-wrap: wrap;
+    }
+
+    .card-wrapper {
+      padding: 0.5rem;
+      flex-basis: 100%;
+    }
+
+    /* your code here 👇 */
+      @media only screen
+  and (min-width: 575px)
+  and (max-width: 766px)
+
+    {
+      .card-wrapper {
+        flex-basis: 50%;
+      }
+    }
+
+      @media only screen
+  and (min-width: 767px)
+
+    {
+      .card-wrapper {
+        flex-basis: calc( 100%/ 3);
+      }
+    }
```

Comment:
Please fix your indentation.

Reply:
THis is still not correct indentation. Please follow the format taught in the documentation.